### DISCUSSION TITLE
Reload which-key without restarting nvim?

### DISCUSSION THREAD

**Discussion by dpetka2001** [LazyVim/LazyVim#91] [Q&A] [answered]
I have made some changes to which-key to add some custom stuff. I would like to know if there
is a way of reloading the module, so that the changes take effect, without restarting nvim every time.
I don't have a problem with the changes taking effect. Everything in my config works fine. Just want 
to know if i can reload the module without restarting. Thanks in advance.

**Comment by folke** [comment, marked as answer]
just restart Neovim

**Reply by dpetka2001** [reply]
That's what I'm doing anyway. Just was wondering if there was another way. Thanks for the answer and 
really congratulations for such an awesome plugin. Just migrated from old config with Plug and I find everything
much faster and also neater thanks to the starter template. Loving it so far :)
